src/pages/Merchant/LoginPage/LoginPage.module.scss

```diff
@@ -0,0 +1,70 @@
+@import "styles/variables";
+
+.login {
+  display: flex;
+  width: 100%;
+  height: 100%;
+  min-height: 100%;
+}
+.loginLeft {
+  display: flex;
+  background-color: $bg-greenish-white-2;
+  flex-direction: column;
+  padding: 60px;
+  width: 50%;
+  height: 100vh;
+  min-height: 100%;
+  align-items: center;
+}
+.loginLeftImage {
+  margin-top: 40px;
+  margin-bottom: 40px;
+  height: 200px;
+  width: 400px;
+}
+.loginRightPanel {
+  width: 50%;
+  padding: 4rem 1.5rem 0rem 1.5rem;
+  display: flex;
+  align-items: center;
+}
+.loginRightPanel-mainBody {
+  border: 1px solid $bg-light-gray;
+  height: 100%;
+  // width: 100%;
+  border-radius: 10px;
+  padding: 35px 45px;
+  display: flex;
+  flex-direction: column;
+  gap: 25px;
+}
+.loginForm {
+  width: 100%;
+}
+.loginRightPanel-inputWrapper {
+  display: flex;
+  flex-direction: column;
+}
+.loginRightPanel-inputWrapper label {
+  font-size: 1rem;
+  font-weight: 700;
+  color: $bg-tertiary;
+}
+.loginRightPanel-inputWrapper input {
+  outline: none;
+  padding: 0.75rem;
+  width: 100%;
+  font-size: 1rem;
+  border: 1px solid $bg-light-gray;
+  border-radius: 7px;
+
+  &:focus {
+    outline: 1px solid $bg-secondary;
+  }
+  &:hover {
+    outline: 1px solid $bg-secondary;
+  }
+  &::placeholder {
+    color: $bg-medium-gray;
+  }
+}
```

Comment:
these styling is already present in main.scss with a class name of basic-input you can dic the class to any element where you want to apply this styling